src/app/components/mnoe-teams-list/mnoe-teams-list.html

```diff
@@ -16,7 +16,7 @@
         <tbody>
           <tr ng-click="$ctrl.toggleTeam(team)" ng-if="!$ctrl.isTeamsLoading" ng-repeat-start="team in $ctrl.teams.list">
             <td>{{team.name}}</td>
-            <td class="text-mute"><span class="badge badge-colour">{{team.users.length}} {{ 'mnoe_admin_panel.dashboard.teams.widget.list.table.show_users' | translate }}</span>
+            <td class="text-mute"><span class="badge badge-colour">{{team.users.length}} {{ 'mnoe_admin_panel.dashboard.teams.widget.list.table.show_users' | translate: { QTY: team.users.length} }}</span>
```

Comment:
Can you interpolate the `QTY` into the string using pluralisation instead of displaying `team.users.length` directly? If it's not possible I'd say it would be good to add `&nbsp;` so it would look like `{{team.users.length}}&nbsp;{{'mnoe_admin_panel...` as white space can sometimes be removed during the frontend build.